compat/mingw.c

```diff
@@ -2400,22 +2400,22 @@ struct hostent *mingw_gethostbyname(const char *host)
 
```

Comment:
The commit message is correct, of course, yet it is not very helpful to understand the motivation behind this commit. If you read Git's commit history, you will find that the commit messages are pretty good at explaining intentions and background information that are not readily available from reading the patch itself, and that is really crucial in particular when future contributors want to touch the same code and need to know a bit more.

See e.g. 6c6ce21baa9b50d394bb8ed9878944504ffd57d8, which not only says what the commit is intended to solve, but also condenses the discussion around the patch into a concise read.

Back to this here commit: from that commit message, a reader would not be able to guess why it is okay to remove the fallback *now*, and what the ramifications are regarding backwards-compatibility. But I think the reader should be able to learn this from the commit's message.